source/Government.cpp

```diff
@@ -144,6 +144,13 @@ void Government::Load(const DataNode &node)
 			enforcementZones.emplace_back(child);
 		else if(key == "provoked on scan")
 			provokedOnScan = true;
+		else if(key == "restricted" && child.HasChildren())
```

Comment:
You put this above `"provoked on scan"` in the `remove` section.

Reply:
Oh... should be below since restricted is after in alphabetical order.